test/packager.test.ts

```diff
@@ -1,20 +1,13 @@
-import { handler } from "../src/packager";
-import { ScheduledEvent } from "aws-lambda";
-import { callback, context } from "./fixtures";
-
-const event: ScheduledEvent = {
-  version: "0",
-  account: "1234567890",
-  region: "us-east-1",
-  detail: {},
-  "detail-type": "Scheduled Event",
-  source: "aws.events",
-  time: "2019-03-01T01:23:45Z",
-  id: "cdc73f9d-aea9-11e3-9d5a-835b769c0d9c",
-  resources: ["arn:aws:events:..."],
-};
+import { handler } from "~/src/packager";
```

Comment:
I added the ability to do this in [`bdb37af`](https://github.com/mbta/ocs_saver/pull/4/commits/bdb37afa12cbb740aa9971b31a5a6cd482431324) — it's not intrinsic to any of the tools we're using. This was because I wanted to avoid things like `import foo from "../../foo"`, because then the import breaks if the directory nesting level of the _importing_ file changes, even though nothing changed about the _imported_ file.

`tsc` sort of lets you do this without a specific "prefix" just by specifying `baseUrl` in the config, but unfortunately that doesn't play nicely with Jest, and in any case I think it's best to have an explicit prefix to make it clear when we're importing relative to the project root.